openstack/swift/etc/statsd-exporter.yaml

```diff
@@ -142,65 +142,3 @@ mappings:
   name: swift_object_replicator
 - match: swift.object-updater.timing
   name: swift_object_updater
-
-  ##############################################################################
-  # extract storage IP from swift-recon metrics (sent by swift-health-statsd)
```

Comment:
That should stay in if there is support for health-statsd and health-export

Reply:
commit reverted